internal/admin/dashboard/templates/page-models.html

```
@@ -177,6 +179,116 @@ <h3 x-text="modelOverrideFormDisplayName || modelOverrideForm.selector || 'Acces
+                        <div class="pricing-override-row">
+                            <div class="form-field pricing-override-type-field">
+                                <label class="form-field-label">Price Type</label>
+                                <select class="form-select" x-model="row.field" data-modal-autofocus>
+                                    <template x-for="option in availablePricingFieldOptions(row)" :key="option.value">
+                                        <option :value="option.value" x-text="option.group + ' - ' + option.label"></option>
+                                    </template>
+                                </select>
+                            </div>
+                            <div class="form-field pricing-override-value-field">
+                                <label class="form-field-label">USD Value</label>
+                                <input type="number" step="any" min="0" inputmode="decimal" x-model="row.value">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)